Pillar_II/esm/src/fesom2/__init__.py

```diff
@@ -68,13 +80,25 @@ def _namelists(start_date: str, config: ConfigParser):
     Returns:
         A dictionary with the name of the namelist, and an optional dictionary of values to process the namelist.
     """
-    model_config = _get_model_config(start_date, config)
+    model_config = _get_model_config(start_date, member, config)
+    temp_min = random.uniform(-1.0, 2.0)
+    temp_max = random.uniform(2.0, 4.0)
+    # N.B.: Salinity must be above 1.
+    sal_min = random.uniform(1.5, 2.5)
+    sal_max = random.uniform(3.0, 4.0)
```

Comment:
Hi @suvarchal 

We tested this and it caused a model blow up.
![Screenshot from 2024-02-22 17-15-47](https://github.com/eflows4hpc/workflow-registry/assets/60807231/d3418f9a-f903-4019-a999-fb6c8bdef9c6)

Link to paper https://gmd.copernicus.org/articles/7/663/2014/gmd-7-663-2014.pdf
[5:17](https://bsc-es.slack.com/archives/D059GDB8HJ6/p1708618662225279)
When the local sea surface salinity is far from
the reference salinity, virtual salt flux formulation can lead to
too small or too large salinity and thus model blowup.

We think this is the problem. However, another run worked but maybe the values were in the range luckily?